docs/src/pages/index.page.tsx

```diff
@@ -25,6 +25,8 @@ import { HomePrimitivePreview } from './HomePrimitivePreview';
 import { Copy } from '@/components/Copy';
 import { Footer } from '@/components/Layout/Footer';
 
+import type { SandpackThemeProp } from '@codesandbox/sandpack-react';
```

Comment:
FYI, we are going to remove `sandpack` in favor of `react-live`. #1395  @dbanksdesign

Reply:
potentially... there are some potential security issues with it since it uses `eval`, still up for debate